group_trigger.go

```diff
@@ -81,22 +82,22 @@ func (tc *TriggerCounter) countNewTriggers(tList *triggerList) error {
 	tc.keyFrame = tList.keyFrame
 	tc.keyTime = tList.keyTime
 	tc.sampleRate = tList.sampleRate
+	if tc.sampleRate <= 0 {
+		// Counting trigger rates
+		return nil
+	}
 	if !tc.initialized {
 		tc.initialize()
 	}
 	for _, frame := range tList.frames {
-		if frame > tc.hi {
+		for frame > tc.hi {
```

Comment:
a comment noting that `tc.hi` will be incremented by `messageAndReset` makes this look less like an infinite loop

Reply:
Good point. I'll do that.